### ISSUE TITLE
Idea: Trim GPX data

### ISSUE THREAD

**Issue by HanSolo68** [gpx-animator/gpx-animator#750]
I have several recordings of travels where the GPX file contains data long before starting and after stopping the ride. Reason is the I turn on my Garmin in the morning and it runs until battery is empty.

Currently I trim the files manually with a text editor which is quite some effort.

It would be nice to have the possibility to define a start data/time and an end date/time for each track. Only data in between should be used.

**Comment by McPringle** [comment]
GPX Animator has a built in functionality to trim start and end of GPX files:
![image](https://github.com/user-attachments/assets/319e7fff-10fe-42e4-b0d7-96b1399f84ae)

**Comment by HanSolo68** [comment]
In the English version I understand that that is the feature, I wanted.

The German translation was too misleading for me.

<img width="809" alt="image" src="https://github.com/user-attachments/assets/d1a76271-1a99-4095-88c6-d3ed1aad1c75">
